bat/RUNASMDEMOS.BAT

```diff
@@ -11,46 +11,53 @@ if /I "%1" == "tron" (set z_TraceMode=tron
 set /A z_NestLevel=%z_NestLevel%+1
 rem ----- Lvl(%z_NestLevel%) Start %0 %1 %2 %3 %4 %5 %6 %7 %8 %9
 
+rem during build on github server the compare steps fail - no clue why.
```

Comment:
Do you want me to investigate reason for fail when running on GitHub?

Reply:
With PR#635 in place, I should be able to diagnose the issue myself. Will have a look when time permits.